snap/snapcraft.yaml

```diff
@@ -216,6 +216,11 @@ parts:
       - --enable-freetype
       - --without-tesseract
       - --datarootdir=/snap/ps-printer-app/current/usr/share/
+    override-pull: |
+      set -eux
+      # Do the actual pull task
+      craftctl default
+      rm -rf gpdl pcl xps
```

Comment:
For Snap it is no problem to do it this way to not build support for undesired input data formats which have extra dependencies (like TIFF), more "elegant" would be to find the correct `./configure` command line options which do the same thing.